### PR TITLE
Release 1.22.1

### LINKED ISSUE #3453: Bump to gnutls 3.6.15-r1 or later in docker build and base image
**Describe the bug**
There is a security vulnerability in the `gnutls` library that has been resolved in the 3.6.15-r1 release.  `gnutls` is used in the alpine base image as well as the [flux dockerfile](https://github.com/fluxcd/flux/blob/21debdeec3012371937348b12434543b196925aa/docker/Dockerfile.flux#L5).  The issue was reported to alpine [here](https://gitlab.alpinelinux.org/alpine/aports/-/issues/12543) and was resolved [before the 3.12.5 release](https://gitlab.alpinelinux.org/alpine/aports/-/commits/3.12-stable) was created.

**Expected behavior**

We should be using reducing security vulnerabilities as much as possible, including bumping to newer versions of packages and newer versions of base images when possible.  We should bump the apline image constraint in the dockerfile to 3.12.5 as well as the version constraint for `gnutls` to `>3.6.15-r1`.

**Additional context**

- Flux version: 1.22.0
- Kubernetes version:
- Git provider:
- Container registry provider:

### PR DESCRIPTION
**Pull request by kingdonb**
Simply fixes CI with #3452, and then building again which rolls the alpine stack.

As of this morning this is enough to mitigate a CVE in #3453 which came from an end-user report. (Fixes #3453)

### CODE REVIEW

**Comment by stefanprodan** [comment]
@kingdonb the CI changes are not relevant to users, I would mention the CVE fix only.

File: chart/flux/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
-appVersion: "1.21.2"
-version: 1.6.2
+appVersion: "1.22.0"
```
Comment (hiddeco) [inline chart/flux/Chart.yaml:2]: Shouldn't this be `1.22.1`, or is this the result of some other dance between release branches?

**Comment by yebyen** [comment]
You are both correct, thanks for catching these issues. I will amend and re-request the review.

File: chart/flux/Chart.yaml (hunk 1 above)
Comment (kingdonb) [reply chart/flux/Chart.yaml:2]: That's right, the update in this branch is housekeeping from the last chart-bump. A new chart-bump branch is ready now and will merge this update once the release is published, in a separate PR.

That will be in #3456, which I just opened as a draft.

**Review by hiddeco — APPROVED** [review]

**Comment by kingdonb** [comment]
Thanks for your responsiveness. I'm going to cut this, and unfortunately push out #3275 to the next release as I haven't had a PR submitted for that yet and so I have had no chance to test the memcached update.

Running up against the end of the day now in the GMT area, and I don't want to delay this release (which would make the dates in the CHANGELOG incorrect, don't want that!)

**Pull request merged**